### DISCUSSION TITLE
Pydantic is all you need: Jason Liu

### DISCUSSION THREAD

**Discussion by MrJarnould** [567-labs/instructor#139] [General]
I found @jxnl's talk at the AI Engineer Summit 2023 helpful and informative. After watching the video, several things "clicked".

Posting it here to share with others:
https://youtu.be/yj-wSRJwrrc

[![Pydantic is all you need](http://i3.ytimg.com/vi/yj-wSRJwrrc/hqdefault.jpg)](https://youtu.be/yj-wSRJwrrc)

**Comment by jxnl** [comment]
any feedback welcome! I'll be making a series with weights and biases in the future to talk more about specifics and see code written
